spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Span.java

```diff
@@ -50,8 +49,6 @@
 	public static final String TRACE_ID_NAME = "X-Trace-Id";
```

Comment:
unrelated note.. I often trip on these constants. I think if the suffix was `HEADER` instead of `NAME` I wouldn't. (assuming they are only used for http headers).